### PR TITLE
Feature/currency conversion

### PR DESCRIPTION
**Pull request by ScuffedNewt**
Allows staff to setup direct currency conversions that users can apply in the bank page
also displays conversions on world page

### CODE REVIEW

File: resources/views/home/_bank_convert.blade.php
Comment (itinerare) [inline resources/views/home/_bank_convert.blade.php:1]: Where is this file used? Implicitly, it's supposed to be included somewhere, but I couldn't spot where.

File: resources/views/home/bank.blade.php [hunk 1]
```diff
@@ -42,6 +42,28 @@
         <a href="{{ url(Auth::user()->url . '/currency-logs') }}">View logs...</a>
     </div>
 
+    <h3>Convert Currency</h3>
```
Comment (itinerare) [inline resources/views/home/bank.blade.php:45]: Maybe only display this section if any currency conversion options exist? Would probably prevent confusion on sites that don't make use of the feature.

**Review by itinerare — CHANGES_REQUESTED** [review]
Seems like a reasonable inclusion given it's not too large and I feel like folk keep wanting something like this, and broadly looks good!

File: resources/views/home/_bank_convert.blade.php
Comment (Ne-wt) [reply resources/views/home/_bank_convert.blade.php:1]: returned from ajax request

File: resources/views/home/bank.blade.php (hunk 1 above)
Comment (Ne-wt) [reply resources/views/home/bank.blade.php:45]: good idea!

File: resources/views/home/_bank_convert.blade.php
Comment (itinerare) [reply resources/views/home/_bank_convert.blade.php:1]: Ahhh gotcha

**Review by itinerare — APPROVED** [review]

**Review by AW0005 — APPROVED** [review]

**Pull request merged**